lib/api/website.js

```diff
@@ -136,6 +138,7 @@ function website(request, log, callback) {
         callback = callbackGetToHead(callback);
     }
     const methodCapitalized = capitalize(request.method);
+    const action = `${request.method.toLowerCase()}Object`;
```

Comment:
I suggest using a regular `if (request.method === 'GET') {...}` etc., it's more readable and maintainable.